buildroot-external/rootfs-overlay/usr/sbin/hassos-supervisor

```diff
@@ -78,6 +67,7 @@ if [ -z "${SUPERVISOR_CONTAINER_ID}" ]; then
         --privileged --security-opt apparmor="hassio-supervisor" \
         -v /var/run/docker.sock:/var/run/docker.sock \
         -v /var/run/dbus:/var/run/dbus \
+        -v /run/supervisor:/run/os:rw \
```

Comment:
@pvizeli  should we maybe rename that to os to avoid confusion? I kinda liked supervisor since then its clear its reserved for the supervisor, but since it is used inside the supervisor container already we cannot use that path inside the container.